examples/exampleproject/modules/optionals/buckets/export_metadata.py

```diff
@@ -2,7 +2,12 @@
 import os
 
 p = os.getenv("AWS_CODESEEDER_NAME")
-d = os.getenv(f"{p.upper()}_DEPLOYMENT_NAME")
-m = os.getenv(f"{p.upper()}_MODULE_NAME")
-file = open("cdk-exports.json")
-print(json.load(file)[f"{p}-{d}-{m}"]["metadata"])
+p_fetch = p.replace("-","_").upper()
+
+d = os.getenv(f"{p_fetch}_DEPLOYMENT_NAME")
+m = os.getenv(f"{p_fetch}_MODULE_NAME")
+cdk_output = open("cdk-exports.json")
+data = json.load(cdk_output)[f"{p}-{d}-{m}"]["metadata"]
+
+with open(f"{p_fetch}_MODULE_METADATA", 'w') as f:
```

Comment:
Does it make sense to use a complete file extension `.json` to make it explicit?

Reply:
@PatWie The file name is a semantic.  In this particular case, this file name `<project>_MODULE_METADATA` is the agreed upon file naming convention between `aws-codeseeder` and `seed-farmer`.  So for the ADDF project, we will look for a file named `ADDF_MODULE_METADATA` with json content.